### ISSUE TITLE
🐛 [firebase_crashlytics] flutter build ipa is infinite since crashlytics 3.1.0

### ISSUE THREAD

**Issue by Francesco-FL** [firebase/flutterfire#11057]
## Bug report

When I run "flutter build ipa" with crashlytics 3.0.17, core 2.8.0 and cloud firestore 4.4.5 it is successful.

When I upgrade crashlytics to 3.1.0, core to 2.9.0, cloud firestore to 4.5.0, then flutter build ipa is infinite, while archiving it stuck at "Run custom shell script '[firebase_crashlytics] Crashlytics Upload Symbols" and not goes further.

### Flutter doctor

Run `flutter doctor` and paste the output below:

<details><summary>Click To Expand</summary>

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.10.2, on macOS 13.3.1 22E772610a darwin-arm64, locale it-IT)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 14.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.1)
[✓] VS Code (version 1.78.2)
[✓] Connected device (2 available)
[✓] Network resources

• No issues found!
```

![Screenshot 2023-05-29 alle 19 02 57](https://github.com/firebase/flutterfire/assets/83022092/f6428027-2011-4b56-9633-f623d5cf8ca0)

**Comment by danagbemava-nc** [comment]
Hi @Francesco-FL, do you have this issue with a new project or is it only with your project? 

Can you share the full logs of `flutter build ipa -v` or the full logs from xcode? 

Please also provide

**Comment by Francesco-FL** [comment]
Thanks for the reply.

The problem is only in my project, not a new one.
I tried to do everything, pod deintegrate, clean build folder, flutter clean, flutterfire configure, but nothing

In the initial post if you click to expand you see the point where the build becomes infinite. (I can't send other logs because they contain personal information)

I have now tried deleting the ios folder and recreating it, and now the archiving is done successfully from Xcode, so problem solved.
